v2/tools/generator/internal/codegen/testdata/ArmResource/Arm_oneof_resource_conversion_on_arm_type_only_azure.golden

```diff
@@ -173,7 +169,16 @@ func (resource *FakeResource) updateValidations() []func(old runtime.Object) (ad
 		func(old runtime.Object) (admission.Warnings, error) {
 			return resource.validateResourceReferences()
 		},
-		resource.validateWriteOnceProperties}
+		resource.validateWriteOnceProperties,
+		func(old runtime.Object) (admission.Warnings, error) {
+			return resource.validateOwnerReference()
+		},
```

Comment:
Does this need to be wrapped? or, could you use `resource.validateOwnerReference` directly?

Reply:
It's wrapped to discard the `old` parameter because it's not needed but I wanted to share a single impl for update/create. Since the signature matches for create it's called directly:
```
// createValidations validates the creation of the resource
func (zone *DnsZone) createValidations() []func() (admission.Warnings, error) {
	return []func() (admission.Warnings, error){zone.validateResourceReferences, zone.validateOwnerReference}
}
```

but for update because the signature doesn't match exactly it's wrapped. This is the same behavior as w/ `validateResourceReferences` so I think it makes sense to do the same thing here. LMK if you see a cleaner way.